src/model/options.model.ts

```diff
@@ -6,5 +6,7 @@ export interface Options {
 
   proxy?: string;
 
+  headers?: string;
```

Comment:
Shouldn't this be `Record<string, string>` or something?

Reply:
Just updated the type, please check again, if it does not work, I need to drive into it at a later moment.